docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '3.7'
+version: '3.0'
```

Comment:
Personally, I'm not very keen on changing this version down to 3.0, only so people don't have to update their docker compose. We started the project a few weeks ago and it was (and is) the ultimate stable version. No one until now had to downgrade this version in the docker-compose.yml to work on the project (except you I think), and even if it were, it wouldn't be necessarily a bad thing.

We are/were 8 people, I think, on this project so everyone can give their opinion so that I ain't the only one

Reply:
I don't see any reason why we should downgrade the docker compose version :thinking: 
Is there an issue caused by this version ?

Reply:
I too think it does not need to downgrade, I did not need to update my docker, it was already updated. But as @lbassin said, if we have a issue because of this version, we can downgrade.

Reply:
My point here is that we don't need `3.7` unless we are using a newer feature that is only in `3.7`. Also this PR is here to ease install and development, so I put `3.0` to be more retro-compatible and minimize things we may need to do before install this application.

Reply:
Restored `3.7`